frontend/packages/ceph-storage-plugin/src/components/create-storage-system/external-storage/ibm-flashsystem/models.ts

```diff
@@ -0,0 +1,13 @@
+import { K8sKind } from '@console/internal/module/k8s';
+
+export const IBMFlashsystemModel: K8sKind = {
+  label: 'IBM Flash System',
+  labelPlural: 'IBM Flash Systems',
```

Comment:
In kind it says flashsystemcluster but in label it is  flash system.
Is that a miss  ?

Reply:
Need to be resolved later. Not blocking PR